charts/adot-exporter-for-eks-on-ec2/values.yaml

```
@@ -331,11 +337,53 @@ adotCollector:
       - dimensions: [[Namespace, ClusterName], [ClusterName]]
         metric_name_selectors:
           - namespace_number_of_running_pods
+    ampreceivers:
+      scrapeInterval: 15s
+      scrapeTimeout: 10s
+      scrapeConfigs: |
+            - job_name: 'k8s_metrics_scrape'
+              sample_limit: 10000
+              metrics_path: /metrics
+              kubernetes_sd_configs:
+                - role: pod
+              relabel_configs:
+                - source_labels: [__meta_kubernetes_pod_annotation_prometheus_io_scrape]
+                  action: keep
+                  regex: true
+                - source_labels: [__meta_kubernetes_pod_annotation_prometheus_io_path]
+                  action: replace
+                  regex: (.+)
+                  target_label: __metrics_path__
+                - source_labels: [__address__]
+                  action: replace
+                  regex: ([^:]+)(?::\d+)?;(\d+)
+                  replacement: $1:$2
```

```suggestion
                  replacement: $$1:$$2
```

The `$` here needs to be doubled to escape from the environment substitution.